core/const.go

```diff
@@ -23,6 +23,9 @@ const (
 	EdgeXService     = "edgex"
 	VaultToken       = "X-Vault-Token"
 
+	// Edgex Microservices
+	Coredata = "coredata"
```

Comment:
I'd suggest using the existing service key for this so services are referred to in a consistent way. See [here](https://github.com/edgexfoundry/go-mod-core-contracts/blob/24a36f294d12b108636e27c6184f22ea24766959/clients/constants.go#L55).

Reply:
This is not for service key, it is for mongodb login user name. Based on the mongodb init script here: https://github.com/edgexfoundry/docker-edgex-mongo/blob/master/init_mongo.js#L101 the name will be "core". If we prefer to keep it same as the original, I will update it as "core".